Resources/en.lproj/Localizable.strings

```diff
@@ -124,6 +124,8 @@
 
 "LOCAL_NETWORK" = "Local Network";
 "CONNECT_TO_SERVER" = "Connect to Server";
+"CONNECT" = "Connect";
+"FILE_SERVER" = "File Servers";
```

Comment:
Do we need to add new strings? For `Connect` could we use `"BUTTON_CONNECT" = "Connect";` ?

Reply:
yes we need to